indexer/mwm_set.hpp

```diff
@@ -80,6 +84,7 @@ class MwmInfo
   platform::LocalCountryFile m_file;  ///< Path to the mwm file.
   atomic<Status> m_status;            ///< Current country status.
   uint32_t m_numRefs;                 ///< Number of active handles.
+  feature::RegionData m_data;
```

Comment:
Please, put this before m_status and m_numRefs - they're quite specific for MwmSet internals and should not be mixed with other fields.